lib/retryChunkLoadPlugin.js

```diff
@@ -52,9 +49,19 @@ if (typeof ${RuntimeGlobals.require} !== "undefined") {
       load();
     });
   };
-}`
-        )
-        return source + script + ';'
+`
+
+        class CustomRuntimeModule extends RuntimeModule {
+          constructor() {
+            super('custom', RuntimeModule.STAGE_BASIC)
```

Comment:
```suggestion
            super(name, RuntimeModule.STAGE_BASIC)
```

Maybe this should also match the plugin name? Not sure if it's a global name

Reply:
yeah nice catch